scripts/test-all.sh

```diff
@@ -105,16 +176,29 @@ echo "=================================================="
 echo "TEST RESULTS SUMMARY:"
 echo "=================================================="
 
+pass_count=0
+fail_count=0
+skip_count=0
+
 for result in "${RESULTS[@]}"; do
     if [[ $result == PASS* ]]; then
-        echo -e "${GREEN}$result${NC}"
+        echo -e "${GREEN}✓ $result${NC}"
+        ((pass_count++))
     elif [[ $result == FAIL* ]]; then
-        echo -e "${RED}$result${NC}"
+        echo -e "${RED}✗ $result${NC}"
+        ((fail_count++))
+    elif [[ $result == SKIP* ]]; then
+        echo -e "${YELLOW}⚠ $result${NC}"
+        ((skip_count++))
     else
         echo -e "${YELLOW}$result${NC}"
     fi
 done
 
+# Show totals
+echo ""
+echo -e "Totals: ${GREEN}$pass_count passed${NC}, ${RED}$fail_count failed${NC}, ${YELLOW}$skip_count skipped${NC}"
```

Comment:
Excellent summary enhancement. The addition of skip counts to the totals provides complete visibility into test execution. The `Totals:` line with color-coded counts makes it easy to assess the overall test health at a glance.